src/index.html

```diff
@@ -9,6 +9,66 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_list_item">
+            <a href="#" class="nav_list_item_link is-active">
```

Comment:
Add some mock anchor links to all nav items
```suggestion
            <a href="#apple" class="nav_list_item_link is-active">
```